app/Http/Controllers/KnowledgeCafe/Library/BookCommentController.php

```diff
@@ -15,8 +15,13 @@ public function store(Request $request, Book $book)
             'user_id' => auth()->id(),
             'comment' => $request->comment,
         ]);
-
         $book->comments()->save($comment);
-        return response()->json();
+        return response()->json($book->comments->last());
```

Comment:
will this make another query? If yes, do you think we can use this instead?
```suggestion
        return response()->json($comment);
```

Reply:
No $book->comments is a collection.